presto-iceberg/pom.xml

```diff
@@ -424,6 +424,7 @@
                             <ignoredClassPattern>module-info</ignoredClassPattern>
                             <ignoredClassPattern>org.apache.avro.*</ignoredClassPattern>
                             <ignoredClassPattern>org.apache.parquet.*</ignoredClassPattern>
+                            <ignoredClassPattern>org.apache.htrace.*</ignoredClassPattern>
```

Comment:
Why is the exclusion required here and other directories ?

Reply:
Without ignoring, the maven duplicate finder plugin complains about "Found duplicate (but equal) classes in [module A, module B]."

Reply:
Does the prersto-hudi bundle contains copy of this class , but not shaded ?

Reply:
Yeah. I just created a PR to shade them https://github.com/apache/hudi/pull/4495

Reply:
When will the new presto-hudi bundle available ? Can you use the new presto-hudi bundle instead of this one ?

Reply:
@arunthirupathi We plan a 0.10.1 minor release in Jan. Can we merge this and we can follow up after? Do you see any issues with this exclusion.

Reply:
> @arunthirupathi We plan a 0.10.1 minor release in Jan. Can we merge this and we can follow up after? Do you see any issues with this exclusion.

I don't see any, if you folks make effort to release this in Jan and remove the exclusion later. As usual, I will revert on first sign of trouble.

Reply:
Once the test passes,  I will merge this change.